.github/actions/setup-java-build/action.yml

```
@@ -13,16 +13,22 @@ inputs:
     description: the location of the custom Maven settings.xml file to install
     default: ".ci.settings.xml"
     required: false
+  cache-key-label:
+    description: the optional label for the maven repository cache key (eg. to distinguish caches for self hosted runners)
+    default: ""
```

```suggestion
    default: "maven"
```

remove `env.KEY_LABEL` completely (don't really seems needed to me), just use `inputs.cache-key-label` directly (altough I will prefer calling it `cache-key-prefix`)